### PR TITLE
Add full node configuration and requirements

### PR DESCRIPTION
**Pull request by pwnfoo**

### CODE REVIEW

File: documentation/operators/src/nodes/nym-api.md [hunk 1]
```diff
@@ -18,31 +18,82 @@ This is important for both the proper decentralisation of the network uptime cal
...
+Make sure your pruning settings are configured as below in `app.toml`:
+
+```
+pruning = "custom"
+
+# This number is likely to be updated once zk-nym signing goes live
+pruning-keep-recent = "750000"
+```
+
+The value of `pruning-interval` can be customised as per your requirement.
```
Comment (mfahampshire) [inline documentation/operators/src/nodes/nym-api.md]: What would an example requirement be here?

File: documentation/operators/src/nodes/nym-api.md [hunk 2]
```diff
@@ -18,31 +18,82 @@ This is important for both the proper decentralisation of the network uptime cal
...
 Operators of Nym API will be rewarded for performing the extra work of taking part in credential generation. These rewards will be calculated **separately** from rewards for block production.
 
 Rewards for credential signing will be calculated hourly, with API operators receiving a proportional amount of the reward pool (333NYM per hour / 237,600 NYM per month), proportional to the percentage of credentials they have signed.
 
-### (Coming Soon) Machine Specs
-We are working on load testing currently in order to get good specs for a full node + Nym API setup. Bear in mind that credential signing is primarily CPU-bound.
+
+### Hardware Requirements
+
+The specification mentioned below is for running a full node alongside the nym-api. It is recommended to run nym-api and a full Nyx node on the same machine for optimum performance.
```
Comment (mfahampshire) [inline documentation/operators/src/nodes/nym-api.md]: missing "`" around "nym-api"

File: documentation/operators/src/nodes/nym-api.md [hunk 3]
```diff
@@ -85,10 +136,10 @@ The API binary currently defaults to running in caching mode. You can run your A
 ./nym-api run
 ```
 
-By default the API will be trying to query a running `nyxd` process (either a validator or RPC node) on `localhost:26657`. This value can be modified either via the `--nyxd-validator ` flag on `run`:
+By default the API will be trying to query your full node running locally on `localhost:26657`. If your node is hosted elsewhere, you can specify the RPC location by using the `--nyxd-validator ` flag on `run`:
```
Comment (mfahampshire) [inline documentation/operators/src/nodes/nym-api.md:140]: maybe we should raise an issue to change the name of this flag, seems a bit misleading considering you're _not_ meant to be hitting a validator with it, but instead a fullnode

Comment (mfahampshire) [reply documentation/operators/src/nodes/nym-api.md:140]: not necessarily related to your PR feel free to ignore, was 'thinking aloud'

**Review by mfahampshire — CHANGES_REQUESTED** [review]

File: documentation/operators/src/nodes/nym-api.md (hunk 3 above)
Comment (pwnfoo) [reply documentation/operators/src/nodes/nym-api.md:140]: Good point!

Comment (pwnfoo) [reply documentation/operators/src/nodes/nym-api.md:140]: Tracked on #4336

**Review by mfahampshire — APPROVED** [review]

**Pull request merged**